test/artefacts/overseas-passports/falkland-islands/applying/adult/afghanistan.txt

```diff
@@ -15,7 +15,9 @@ Don’t book travel until you have a valid passport. If you need to travel more
 
 ## Cost
 
-You’ll have to pay a fee for your passport and a courier fee of £19.86. The courier fee pays for your passport and supporting documents to be sent back to you securely.
+You’ll have to pay a fee for your passport and a courier fee of £19.86.
+
+The courier fee pays for your passport and supporting documents to be sent back to you securely.
```

Comment:
We're fairly confident that this is because the original changes were related to replacing a passport, and these are related to applying for a passport.